test/algconvergence/ode_convergence_tests.jl

```diff
@@ -46,6 +46,15 @@ testTol = 0.2
     sim3 = test_convergence(dts, prob, RKM())
     @test sim3.𝒪est[:l∞]≈4 atol=0.2
 
+    sim_ps6 = test_convergence(dts2, prob, PSRK4p7q6())
+    @test sim_ps6.𝒪est[:l∞]≈4 atol=testTol
+
+    sim_ps5 = test_convergence(dts2, prob, PSRK3p6q5())
+    @test sim_ps5.𝒪est[:l∞]≈4 atol=testTol
+
+    sim_ps4 = test_convergence(dts2, prob, PSRK3p5q4())
+    @test sim_ps4.𝒪est[:l∞]≈4 atol=testTol
```

Comment:
```suggestion
    let alg = PSRK4p7q6()
        sim_ps6 = test_convergence(dts2, prob, alg)
        @test sim_ps6.𝒪est[:l∞]≈alg_order(alg) atol=testTol
    end

    let alg = PSRK3p6q5()
        sim_ps5 = test_convergence(dts2, prob, alg)
        @test sim_ps5.𝒪est[:l∞]≈alg_order(alg) atol=testTol
    end

    let alg = PSRK3p5q4()
        sim_ps4 = test_convergence(dts2, prob, alg)
        @test sim_ps4.𝒪est[:l∞]≈alg_order(alg) atol=testTol
    end
```

Reply:
To clarify visually where one test begins/ends. I don't have a strong preference whether there is a `let` block or not. But we would increase the test coverage by using `alg_order(alg)` here.